man/systemd.network.xml

```diff
@@ -2969,12 +2986,25 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             analogous to the <varname>BitRate=</varname> and <varname>SamplePoint=</varname> keys.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>DataTimeQuanta=</varname></term>
```

Comment:
suffix with NSec=?

we usually suffix time values with their unit, and here it is particular relevant, since it's nsec rather than our usual usec